pipeline_dp/__init__.py

```diff
@@ -1,4 +1,4 @@
-from pipeline_dp.budget_accounting import BudgetAccountant
```

Comment:
Nit: Are we sure we want to reference the base class only through the `pipeline_dp.budget_accounting`?  
If someone creates a new `BudgetAccountant`, they gotta import `budget_accounting`, whereas with `{Naive,PLD}BudgetAccountant` they can just do:
```python
import pipeline_dp as pldp
from pipeline_dp import budget_accountant


def do_something_special(accountant: budget_accounting.BudgetAccountant):
    # Also note how we repeat ourselves with "budget_accounting.BudgetAccountant"...
    pass

accountant = pldp.PLDBudgetAccountant(params)
do_something_special(accountant)
```

Reply:
Thanks, a good point! A base abstract class of a public API class is also a part of the public API.

 I've returned back import of `BudgetAccountant`.